scripts/sync_org_skills.py

```
@@ -0,0 +1,68 @@
+        TARGET_DIR.mkdir(parents=True, exist_ok=True)
+
+        copied_count = 0
+        for item in ORG_SKILLS:
+            src = skills_root / item
+            dst = TARGET_DIR / item
+            if src.exists():
+                if src.is_dir():
+                    shutil.copytree(src, dst)
+                else:
+                    shutil.copy2(src, dst)
+                logger.info(f"Copied {item} -> org-wide-skills/{item}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)